src/app/workflow/launch-third-party/launch-third-party.component.ts

```diff
@@ -102,8 +102,6 @@ export class LaunchThirdPartyComponent extends Base implements OnChanges, OnInit
               private descriptorsQuery: DescriptorsQuery,
               private descriptorsService: DescriptorsService) {
     super();
-    iconRegistry.addSvgIcon('firecloud',
```

Comment:
Since you removed this, you can also remove the file `FireCloud-white-icon.svg`.